docs/src/main/sphinx/sql/alter-table.rst

```diff
@@ -39,6 +39,11 @@ The ``ALTER TABLE EXECUTE`` statement followed by a ``command`` and
 parameters. ``ALTER TABLE EXECUTE`` supports different commands on a
 per-connector basis.
 
+Note that ``=>`` is the keyword used for passing parameters to a specified command.
```

Comment:
`=>` is not a word, so cannot be a keyword. "operator"? "syntax"?

@martint may know

Reply:
symbol ?